frontend/pages/hosts/details/cards/MunkiIssues/MunkiIssuesTableConfig.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { capitalize } from "lodash";
+import ReactTooltip from "react-tooltip";
+
+import formatDistanceToNowStrict from "date-fns/formatDistanceToNowStrict";
+import { abbreviateTimeUnits } from "utilities/helpers";
+
+import PATHS from "router/paths";
+import HeaderCell from "components/TableContainer/DataTable/HeaderCell/HeaderCell";
+import TextCell from "components/TableContainer/DataTable/TextCell";
+import TruncatedTextCell from "components/TableContainer/DataTable/TruncatedTextCell";
+import TooltipWrapper from "components/TooltipWrapper";
+import { IMunkiIssue } from "interfaces/host";
+
+interface IHeaderProps {
+  column: {
+    title: string;
+    isSortedDesc: boolean;
+  };
+}
+interface ICellProps {
+  cell: {
+    value: number | string | string[];
+  };
+  row: {
+    original: IMunkiIssue;
+    index: number;
+  };
+}
+
+interface IStringCellProps extends ICellProps {
+  cell: {
+    value: string;
+  };
+}
+
+interface IDataColumn {
+  title: string;
+  Header: ((props: IHeaderProps) => JSX.Element) | string;
+  accessor: string;
+  Cell: (props: IStringCellProps) => JSX.Element;
+  disableHidden?: boolean;
+  disableSortBy?: boolean;
+  disableGlobalFilter?: boolean;
+  sortType?: string;
+  // Filter can be used by react-table to render a filter input inside the column header
+  Filter?: () => null | JSX.Element;
+  filter?: string; // one of the enumerated `filterTypes` for react-table
+  // (see https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js)
+  // or one of the custom `filterTypes` defined for the `useTable` instance (see `DataTable`)
+}
+
+interface IMunkiIssueTableData extends IMunkiIssue {
+  time: string;
+}
+
+// NOTE: cellProps come from react-table
+// more info here https://react-table.tanstack.com/docs/api/useTable#cell-properties
+export const generateMunkiIssuesTableHeaders = (): IDataColumn[] => {
+  const tableHeaders: IDataColumn[] = [
```

Comment:
I think you could just do `export const munkiIssuresTableHeaders` and do away with this generate function as well.

Reply:
100